snuba/api.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from datetime import datetime, timedelta
-from flask import Flask, redirect, render_template, request
+from flask import Flask, redirect, render_template, request as http_request
```

Comment:
Alternatively, could import this as `flask.request` which might be a little clearer?

Once query processing is less coupled to HTTP responses, we can move all the query stuff out of this `api` module and go back to `request`.